### PR TITLE
Add support for Python 3.9

### LINKED ISSUE #41: Add support for Python 3.9

### PR DESCRIPTION
**Pull request by fealho**
To add support for python 3.9 the version requirements of `torch`, `tqdm`, `pandas` and `numpy` had to be updated.

Resolve #41.

### CODE REVIEW

**Review by katxiao — APPROVED** [review]
lgtm

File: conda/meta.yaml [hunk 1]
```diff
@@ -17,16 +17,16 @@ requirements:
...
+    - pytorch >=1.7.1,<2
     - tqdm >=4.10,<5
   run:
-    - python >=3.6,<3.9
-    - numpy >=1.18.0,<2
-    - pandas >=1.1,<1.1.5
-    - pytorch >=1.4,<2
+    - python >=3.6,<3.10
+    - numpy >=1.20.0,<2
+    - pandas >=1.1.3,<1.1.5
```
Comment (amontanez24) [inline conda/meta.yaml:28]: Let's update the upper bound to 2

File: tasks.py [hunk 2]
```diff
@@ -28,6 +37,30 @@ def coverage(c):
...
+
+
+def _validate_python_version(line):
+    python_version_match = re.search(r"python_version(<=?|>=?)\'(\d\.?)+\'", line)
+    if python_version_match:
+        python_version = python_version_match.group(0)
+        comparison = re.search(r'(>=?|<=?)', python_version).group(0)
+        version_number = python_version.split(comparison)[-1].replace("'", "")
+        comparison_function = COMPARISONS[comparison]
+        return comparison_function(platform.python_version(), version_number)
```
Comment (amontanez24) [inline tasks.py:59]: Why is this added twice?

**Review by amontanez24 — CHANGES_REQUESTED** [review]
Couple of simple fixes to make before approval!

File: conda/meta.yaml [hunk 3]
```diff
@@ -17,16 +17,16 @@ requirements:
   host:
     - pip
     - pytest-runner
-    - python >=3.6,<3.9
+    - python >=3.6,<3.10
     - numpy >=1.18.0,<2
-    - pandas >=1.1,<1.1.5
-    - pytorch >=1.4,<2
+    - pandas >=1.1.3,<1.1.5
+    - pytorch >=1.7.1,<2
```
Comment (amontanez24) [inline conda/meta.yaml:23]: In the spreadsheet we have the minimum as `1.4`. Why did we need it to be `1.7.1` and should it be updated for all the libraries?

Comment (fealho) [reply conda/meta.yaml:23]: Yes, I believe it does: https://download.pytorch.org/whl/torch_stable.html

File: tasks.py (hunk 2 above)
Comment (fealho) [reply tasks.py]: Oops 😅

**Review by amontanez24 — APPROVED** [review]
